sdk/servicebus/service-bus/src/core/managementClient.ts

```diff
@@ -45,17 +45,16 @@ import {
 import { max32BitNumber } from "../util/constants";
 import { Buffer } from "buffer";
 import { OperationOptionsBase } from "./../modelsToBeSharedWithEventHubs";
-import { AbortSignalLike } from "@azure/abort-controller";
 import { ReceiveMode } from "../models";
 import { translateServiceBusError } from "../serviceBusError";
 import { defaultDataTransformer } from "../dataTransformer";
 
 /**
  * @internal
  */
-export interface SendManagementRequestOptions extends SendRequestOptions {
+export interface SendManagementRequestOptions extends SendRequestOptions, OperationOptionsBase {
   /**
-   * The name of the sender or receiver link associated with the managmenet operations.
+   * The name of the sender or receiver link associated with the management operations.
```

Comment:
I think you're our best proofreader.

Reply:
I use the spell check extension in VSCode. 😁

Reply:
FINE.